pkg/dockerfile/dockerfile.go

```diff
@@ -38,6 +42,7 @@ import (
 func Stages(opts *config.KanikoOptions) ([]config.KanikoStage, error) {
 	var err error
 	var d []uint8
+	lintDockerfilePath := constants.DockerfilePath
```

Comment:
could you rename to `dockerFilePath`? at first when reading the code I assumed there was a separate lintDockerFile that was different from the DockerFile passed in. It's a bit confusing.